www/scipylib/building/windows.rst

```diff
@@ -5,374 +5,221 @@ Building From Source on Windows
-    lapack.lib(xerbla.o) : error LNK2019: unresolved external symbol _s_stop referenced in function _xerbla_
-    lapack.lib(ilaenv.o) : error LNK2019: unresolved external symbol _s_cmp referenced in function _ilaenv_
-    lapack.lib(ilaenv.o) : error LNK2019: unresolved external symbol _s_copy referenced in function _ilaenv_
-    lapack.lib(zlahqr.o) : error LNK2019: unresolved external symbol _z_abs referenced in function _zlahqr_
-    lapack.lib(zlanhe.o) : error LNK2019: unresolved external symbol _z_abs referenced in function _zlanhe_
-    lapack.lib(zgebal.o) : error LNK2019: unresolved external symbol _z_abs referenced in function _zgebal_
-    lapack.lib(zlange.o) : error LNK2019: unresolved external symbol _z_abs referenced in function _zlange_
-    lapack.lib(zlanhs.o) : error LNK2019: unresolved external symbol _z_abs referenced in function _zlanhs_
-    lapack.lib(zhseqr.o) : error LNK2019: unresolved external symbol __alloca referenced in function _zhseqr_
-    lapack.lib(zlarfx.o) : error LNK2019: unresolved external symbol __alloca referenced in function _zlarfx_
-    lapack.lib(zlahqr.o) : error LNK2019: unresolved external symbol _z_sqrt referenced in function _zlahqr_
-    build\lib.win32-2.4\numpy\linalg\lapack_lite.pyd : fatal error LNK1120: 10 unresolved externals
-
-you need to add the g2c and gcc libraries to the ATLAS and LAPACK
-libraries you have already. With Cygwin, you can find these in
-``/lib/gcc/i686-pc-mingw32/3.4.4``. Copy them to ``g2c.lib`` and ``gcc.lib``,
-respectively, and modify ``site.cfg`` accordingly. 
-
-
-Building SciPy
---------------
-
-.. _ATLAS: http://math-atlas.sourceforge.net/
-.. _SSE3 problems: http://math-atlas.sourceforge.net/errata.html#sse3kill
-.. _CPU-Z: http://www.cpuid.com/cpuz.php
-
+Up to this point, we have been discussing optimized versions of BLAS_ coupled with LAPACK_. It goes without saying then
+that it must be possible then to build NumPy and SciPy with an unoptimized BLAS_ library. Pre-built libraries are readily
+available here, though **be sure to check the environment in which the libraries** were built in. Otherwise, NumPy and
+SciPy will not build. However, if none of the environments match your own environment, the libraries themselves can be
+downloaded as ZIP files by searching for a "download" section on the BLAS_ and LAPACK_ webpages. Rough installation
+instructions can be found `here <http://ab-initio.mit.edu/wiki/index.php/Template:Installing_BLAS_and_LAPACK>`__ for
+BLAS_ and on the LAPACK_ homepage for LAPACK_. While these instructions are for Linux, you should be able to follow these
+instructions fairly well if you have either Cygwin_ or Mingw-w64_ installed on your computer.
+
+Finally, it goes without saying that because this is an unoptimized BLAS_ library, do not expect very good performance
+with either NumPy or SciPy. Thus, if performance is of the utmost importance, do read the previous sections in order to
```

Comment:
A fresh look suggests that that whole last section is not necessary.  I mean I even started it by saying "it goes without saying..." before proceeding to say it :smile: